### PR TITLE
Runners savage damage change.

### PR DESCRIPTION
**Pull request by David-Stormwell**
## About The Pull Request

Makes the bonus damage from savage 15 less damage.
What it was before: use_plasma(extra_dam * 2),
now: use_plasma(extra_dam * 1.5).

## Why It's Good For The Game

As it stands, currently runner is quite overturned, being able to wipe out squads solo of even higher tier players without as much as breaking a sweat. As even a Rav seems midly weak compared to a the staggering amount of damage savage gives, which is currently 72, which via my PR changes to about 57 being the maximum, tested in-game as well, so, all in all, makes the runner still quite effective, but less free damage from a single pounce at the very least.

## Changelog
:cl:
balance: rebalanced runner's savage to be about 15 less damage on savage.
/:cl:

### CODE REVIEW

**Comment by Naaanii** [comment]
Muh rouny...

**Comment by usnpeepoo** [comment]
Erm since when could runners (t1 caste) solo squads. The most they can do is pounce one guy near the front for a combo with another xeno, the savage damage is only good for soloing people. Nerf out of nowhere for a t1 caste that’s overturned for just doing damage on a pounce? Runners have no other stun, a few secs of damage invulnerability and that’s it. Savage is the only thing it has going for it.

**Comment by vilereaver41** [comment]
needed. over performing T1 caste, id rather this get nerfed and hunter get fixed

**Comment by David-Stormwell** [comment]
> Erm since when could runners (t1 caste) solo squads. The most they can do is pounce one guy near the front for a combo with another xeno, the savage damage is only good for soloing people. Nerf out of nowhere for a t1 caste that’s overturned for just doing damage on a pounce? Runners have no other stun, a few secs of damage invulnerability and that’s it. Savage is the only thing it has going for it.

I'll make several videos and ask runner mains if they do.
_quick answer they do._

**Comment by Lefunny** [comment]
> > Erm since when could runners (t1 caste) solo squads. The most they can do is pounce one guy near the front for a combo with another xeno, the savage damage is only good for soloing people. Nerf out of nowhere for a t1 caste that’s overturned for just doing damage on a pounce? Runners have no other stun, a few secs of damage invulnerability and that’s it. Savage is the only thing it has going for it.
> 
> I'll make several videos and ask runner mains if they do. _quick answer they do._

They can even do but prob nothing with the pounce savage as its is literal a skill with gets only one marine

File: code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm [hunk 1]
```diff
@@ -52,7 +52,7 @@
 	GLOB.round_statistics.runner_savage_attacks++
 	SSblackbox.record_feedback("tally", "round_statistics", 1, "runner_savage_attacks")
 	M.attack_alien_harm(src, extra_dam, FALSE, TRUE, TRUE, TRUE) //Inflict a free attack on pounce that deals +1 extra damage per 4 plasma stored, up to 35 or twice the max damage of an Ancient Runner attack.
-	use_plasma(extra_dam * 2)
+	use_plasma(extra_dam * 1.5)
 	savage_used = TRUE
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm:56]: like i said in disc you accidentally buffed it

**Pull request closed without merging**